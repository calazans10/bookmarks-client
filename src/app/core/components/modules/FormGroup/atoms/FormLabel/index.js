import React from 'react';
import PropTypes from 'prop-types';

const FormLabel = ({ htmlFor, children }) => (
  <label className="form-group__label" htmlFor={htmlFor}>
    {children}
  </label>
);

FormLabel.propTypes = {
  htmlFor: PropTypes.string.isRequired,
  children: PropTypes.node.isRequired,
};

export default FormLabel;
