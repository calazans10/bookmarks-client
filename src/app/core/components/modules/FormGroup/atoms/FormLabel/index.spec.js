import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import FormLabel from './index';

describe('FormLabel', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<FormLabel htmlFor="test">A label</FormLabel>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
