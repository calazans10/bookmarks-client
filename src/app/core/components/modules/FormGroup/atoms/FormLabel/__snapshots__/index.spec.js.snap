// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormLabel renders without crashing 1`] = `
<label
  className="form-group__label"
  htmlFor="test"
>
  A label
</label>
`;
